taqueria-plugin-jest/proxy.ts

```diff
@@ -70,8 +70,8 @@ const createPartition = async (partitionDir: SanitizedAbsPath.t, projectDir: San
 					writeFile(
 						partitionCfgAbsPath,
 						toPartitionCfg(
-							getPartitionAbspath(partitionDir),
-							getRootConfigAbspath(projectDir),
+							SanitizedPath.create(relative(partitionDir, partitionDir)),
```

Comment:
Why are there two identical `partitionDir` as arguments for `relative()`?

Reply:
It's the relative path from `partitionDir` to itself